python/ray/tests/test_nccl_channel.py

```diff
@@ -341,12 +341,10 @@ def test_direct_return(ray_start_cluster):
     chan_typ.set_nccl_group_id(nccl_id)
     receiver_to_node = [(receiver, get_actor_node_id(receiver))]
     sender.create_traced_channel.remote("tensor_metadata", receiver_to_node)
-    sender.create_traced_channel.remote("cpu_data", receiver_to_node)
```

Comment:
In this case, the cpu_data_channel is not None, and direct_return is True. This is invalid.